### PR TITLE
Addition of resource and data source for vnsRedirectHealthGroup

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

File: examples/redirect_health_group/main.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+resource "aci_tenant" "terraform_tenant" {
+  name        = "tf_tenant"
+  description = "This tenant is created by terraform"
+}
+
+resource "aci_l4_l7_redirect_health_group" "example" {
+  tenant_dn  = aci_tenant.terraform_tenant.id
+  name       = "example"
+}
```
Comment (lhercot) [inline examples/redirect_health_group/main.tf:24]: Can you also add this section in the example of aci_destination_of_redirected_traffic ?

File: website/docs/r/redirect_health_group.html.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  tenant_dn  = aci_tenant.example.id
+  name       = "example"
+}
+```
+
+## Argument Reference ##
+
+* `tenant_dn` - (Required) Distinguished name of the parent Tenant object.
+* `name` - (Required) Name of the L4-L7 Redirect Health Group object.
+* `annotation` - (Optional) Annotation of the L4-L7 Redirect Health Group object.
```
Comment (lhercot) [inline website/docs/r/redirect_health_group.html.markdown:36]: where is the description and name_alias?

**Review by lhercot — CHANGES_REQUESTED** [review]

File: website/docs/d/redirect_health_group.html.markdown [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+subcategory: "Tenant Policies"
```
[the remark this answers is not in the document]
Comment (shrsr) [reply website/docs/d/redirect_health_group.html.markdown:2]: ok

File: website/docs/r/redirect_health_group.html.markdown [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+subcategory: "Tenant Policies"
```
[the remark this answers is not in the document]
Comment (shrsr) [reply website/docs/r/redirect_health_group.html.markdown:2]: ok

File: aci/resource_aci_vnsredirecthealthgroup.go [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+}
+
+func getRemoteL4L7RedirectHealthGroup(client *client.Client, dn string) (*models.L4L7RedirectHealthGroup, error) {
+	vnsRedirectHealthGroupCont, err := client.Get(dn)
+	if err != nil {
+		return nil, err
+	}
+	vnsRedirectHealthGroup := models.L4L7RedirectHealthGroupFromContainer(vnsRedirectHealthGroupCont)
+	if vnsRedirectHealthGroup.DistinguishedName == "" {
+		return nil, fmt.Errorf("L4 L7Redirect Health Group %s not found", dn)
```
Comment (lhercot) [inline aci/resource_aci_vnsredirecthealthgroup.go:48]: L4-L7 Redirect Health Group %s not found

File: aci/resource_aci_vnsredirecthealthgroup.go [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+	d.Set("description", vnsRedirectHealthGroup.Description)
+	vnsRedirectHealthGroupMap, err := vnsRedirectHealthGroup.ToMap()
+	if err != nil {
+		return d, err
+	}
+	dn := d.Id()
+	if dn != vnsRedirectHealthGroup.DistinguishedName {
+		d.Set("tenant_dn", "")
+	} else {
+		d.Set("tenant_dn", strings.Split(GetParentDn(vnsRedirectHealthGroup.DistinguishedName, fmt.Sprintf("/"+models.RnvnsRedirectHealthGroup, vnsRedirectHealthGroupMap["name"])), "/svcCont")[0])
```
Comment (lhercot) [inline aci/resource_aci_vnsredirecthealthgroup.go:64]: if you are splitting on /svcCont, you probably don't need the GetParentDn. You can either use split or use GetParentDn twice.

File: examples/redirect_health_group/main.tf [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    aci = {
+      source = "ciscodevnet/aci"
+    }
+  }
+}
+
+provider "aci" {
+  username = "" # <APIC username>
+  password = "" # <APIC pwd>
+  url      = "" # <cloud APIC URL>
```
Comment (lhercot) [inline examples/redirect_health_group/main.tf:12]: APIC URL (remove cloud)

**Review by lhercot — CHANGES_REQUESTED** [review]

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**